src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -229,11 +250,20 @@ const SampleChatInfoScreen = () => {
     if (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER) {
       return (
         <View style={styles.mr7}>
-          <AnonymousIcon
-            color={anon_user_info_color_code}
-            emojiCode={anon_user_info_emoji_code}
-            size={normalize(48)}
-          />
+          {betterSocialMember &&
+          item.user_id === betterSocialMember[memberChat?.user_id]?.user_id ? (
+            <AnonymousIcon
+              color={betterSocialMember[memberChat?.user_id]?.anon_user_info_color_code}
+              emojiCode={betterSocialMember[memberChat?.user_id]?.anon_user_info_emoji_code}
+              size={normalize(48)}
+            />
+          ) : (
+            <AnonymousIcon
+              color={anon_user_info_color_code}
+              emojiCode={anon_user_info_emoji_code}
+              size={normalize(48)}
+            />
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the `AnonymousIcon` based on whether the user is a better social member or not improves the customization of the user interface. However, make sure that the `betterSocialMember` object and its properties are always defined to avoid potential runtime errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->